tests/test_make_graph.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from conftest import FakeLazyJson
+
+from conda_forge_tick.lazy_json_backends import LazyJson
+from conda_forge_tick.make_graph import try_load_feedstock
+
+
+@pytest.mark.parametrize("container_enabled", [True, False])
+@pytest.mark.parametrize("mark_not_archived", [True, False])
+def test_try_load_feedstock(
+    request: pytest.FixtureRequest, mark_not_archived: bool, container_enabled: bool
+):
+    if container_enabled:
+        request.getfixturevalue("use_containers")
+
+    feedstock = "typst-test"  # archived
+
+    fake_lazy_json = FakeLazyJson()  # empty dict
+
+    with fake_lazy_json as loaded_lazy_json:
+        # FakeLazyJson is not an instance of LazyJson
+        # noinspection PyTypeChecker
+        data = try_load_feedstock(feedstock, loaded_lazy_json, mark_not_archived).data  # type: ignore
+
+    if mark_not_archived:
+        assert data["archived"] is False
+    else:
+        # apparently, this is how the function behaves
+        assert "archived" not in data
```

Comment:
So the feedstock node attrs are not stateless. The graph data will have an "archived" entry sometimes if the bot detects an archived feedstock on the fly. So the mark not archived keyword is there to reset existing feedstocks that are marked as archived. So this test function should set archived to true and ensure it gets reset.

Reply:
Like this?

abf23f841342cf6b2158ce60005ce306f600bb9c